bftengine/src/bftengine/PersistentStorageDescriptors.cpp

```diff
@@ -0,0 +1,276 @@
+// Concord
+//
+// Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+
+#include "PersistentStorageDescriptors.hpp"
+
+using namespace std;
+
+namespace bftEngine {
+namespace impl {
+
+/***** DescriptorOfLastExitFromView *****/
+
+DescriptorOfLastExitFromView::DescriptorOfLastExitFromView() {
+  ViewsManager::PrevViewInfo element;
+  for (uint32_t i = 0; i < kWorkWindowSize; ++i) {
```

Comment:
In this case, the "elements" vector can be empty

Reply:
Agree.